### ISSUE TITLE
whenever i rightclick the aimbot moves in a single direction and the screen capture freezes.

### ISSUE THREAD

**Issue by bomerman234** [zeyad-mansour/Lunar#92]
The aimbot moves past the target and keeps moving the mouse without stopping whenever i try to aim.

**Comment by taskinberkay** [comment]
Yes. I am having the same issue. I think it is caused by an infinite loop happening after move_crosshair function is called.

**Comment by wnfxsa1213** [comment]
So ，how to solve this problem

**Comment by pwengiun** [comment]
same issue

**Comment by Spo3sk** [comment]
same here

**Comment by Spo3sk** [comment]
> Yes. I am having the same issue. I think it is caused by an infinite loop happening after move_crosshair function is called.

but do u know how can we fix it?
